qtconsole/rich_jupyter_widget.py

```diff
@@ -10,9 +10,10 @@
 
 from ipython_genutils.path import ensure_dir_exists
 from traitlets import Bool
+from pygments.util import ClassNotFound
```

Comment:
```suggestion
from pygments.util import ClassNotFound

```

This needs an empty line to separate third-party imports from local ones.